cypress/e2e/PageObjects/dashboardPO.cy.js

```diff
@@ -166,21 +167,67 @@ function CloseModalButton() {
   return cy.get("[data-cy ='x-button']")
 }
 
-function validateExitBetaModalbuttonLink() {
+function getcardNumber() {
+  var a = []
+  var NumberOfCards
+  dashboardData.forEach((card) => {
+    NumberOfCards = a.push(card)
+  })
+  return NumberOfCards
+}
+
+function validateExitBetaModalbuttonLink(SectionName, LinkName) {
   return cy
-    .get("[data-cy ='cards']")
-    .find('li>a')
+    .get('[data-cy="sectionList"]')
+    .find('div>div')
     .each(($el1, index, $list) => {
-      cy.wrap($el1).click()
-      ExitBetaModal().should('be.visible')
-      StayOnBetabutton().click()
-      cy.wrap($el1).click()
-      ExitBetaModal()
-        .find('a')
-        .should('have.length', '1')
-        .and('not.have.length', 0)
-        .and('not.have.attr', 'href', '#undefined')
-      CloseModalButton().click()
+      const heading = $el1.find('h3')
+
+      if (heading.text() == SectionName) {
```

Comment:
Triple equals is always better than double equals (Best practice)

`===` > `==`

Reply:
done  :)